src/app.ts

```diff
@@ -6,11 +6,18 @@ import { exampleConfiguration } from './configuration';
  * Create widget on the document
  */
 function createWidget(): void {
-  const form = new Form();
-  const notion = new Notion(exampleConfiguration);
+  const widget = new Form(exampleConfiguration.form);
+  const notion = new Notion(exampleConfiguration.notion);
 
-  form.container.addEventListener('click', () => {
-    notion.send('Hello');
+  widget.form.addEventListener('submit', (event) => {
```

Comment:
maybe make `form` a  private field of Form class and just pass onSubmit callback to constructor?